CODE_OF_CONDUCT.md

```diff
@@ -60,7 +60,7 @@ representative at an online or offline event.
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
 reported to the community leaders responsible for enforcement at
```

Comment:
**suggestion:** How does this sound to you? We already have the list of maintainers in the [README
](https://github.com/rouge-ruby/rouge#maintainers).

```suggestion
reported by contacting one of the project maintainers.
```

Reply:
Thank you for your thoughtful feedback @dradetsky 👍🏼 Really appreciated it.

> since it's extremely easy for us to underestimate the difficulty of these.

Yep, totally agree with this. Sorry for the oversight.

We could perhaps reword it to read like the following:

```
reported to the project maintainers responsible for enforcement at jneen@jneen.net or tan.le@hey.com.
```

What do you think?

Reply:
10/10. I laughed, I cried, I gave it 2 thumbs up.